src/freshContent/settings/settings.html

```diff
@@ -1,23 +1,25 @@
 <!DOCTYPE html>
-<html data-preload>
+<html lang="de" data-preload>
 
 <head>
   <meta charset="UTF-8">
-  <script src="settings.js"></script>
-  <link rel="stylesheet" href="/css/components/switch.css">
-  <link rel="stylesheet" href="/css/settings.css">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TUFast - Einstellungen</title>
+  <script defer src="settings.js"></script>
```

Comment:
defer nötig?

Reply:
Mindestens empfehlenswert, wir haben in dem Script viele DOM Zugriffe, die im schlimmsten Fall mal fehlschlagen könnten, wenn das Script vor dem DOM ausgeführt wird. Scheint zwar bisher keine Probleme verursacht zu haben, aber schaden tut es in jedem Fall nicht :)

Reply:
ok